app/models/unpublishing.rb

```diff
@@ -3,10 +3,17 @@ class Unpublishing < ActiveRecord::Base
 
   belongs_to :content_item
 
+  VALID_TYPES = %w(
+    gone
+    withdrawal
+    redirect
+    substitute
+  ).freeze
```

Comment:
In the docs:

> Can be "gone", "withdrawal", or "redirect".

I presume that "substitute" is an internal-only type? Do we expose it back to publishing apps if they make get requests to other endpoints?